gstreamer/src/message.rs

```diff
@@ -1171,13 +1171,7 @@ impl<'a> MessageBuilder<'a> {
         }
     }
 
-    fn clone_and_chain(&self, other_fields: &[(&'a str, &'a ToSendValue)]) -> Self {
-        Self {
-            other_fields: self.other_fields.iter().cloned()
-                .chain(other_fields.iter().cloned())
-                .collect()
-        }
-    }
+    // TODO: restore clone_and_chain method and condition it to the "v1_14" feature
```

Comment:
I would just comment it out for now instead of removing. That way adding it back in a few weeks is going to be less annoying :)